dt-login/pages/base.php

```diff
@@ -30,11 +30,11 @@ public function _header(){
     }
     public function header_style(){
         ?>
+
         <style>
-            body {
-                background: white;
-            }
+            <?php require __DIR__ . '/../../dt-assets/build/css/style.min.css' ?>
```

Comment:
@squigglybob 
this includes ALL of the 30 kB of D.T css in the header.
It is just for this UI, right:
![image](https://github.com/DiscipleTools/disciple-tools-theme/assets/24901539/5610e001-c164-41a0-95cf-d9d5a3b362f6)

Reply:
I could sift the CSS, for what is needed for this/ write some CSS to make it look like this 😆

Reply:
I've created a stripped down css for the login page, that has no dependencies on foundation either.

I wasn't able to test the page where you create a new password after receiving the lost password email. 😬 
it was taking me to the wp-login new password page instead